lib/ui/src/modules/ui/components/search_box.js

```diff
@@ -61,11 +70,18 @@ export default class SearchBox extends React.Component {
     this.fireOnKind = this.fireOnKind.bind(this);
   }
 
+  componentDidUpdate(prevProps) {
+    // remove current focus on opening to prevent firing 'enter' keyDowns on it when modal closes
+    if (this.props.showSearchBox && !prevProps.showSearchBox && document.activeElement) {
+      document.activeElement.blur();
+    }
```

Comment:
This can be removed if and when https://github.com/reactjs/react-modal/issues/464 resolves